core/src/main/java/org/jclouds/osgi/Bundles.java

```diff
@@ -131,6 +131,6 @@ public T apply(Class<?> in) {
    }
 
    private static Iterable<String> splitOrEmptyAndClose(InputStream in) throws IOException {
-      return Splitter.on('\n').omitEmptyStrings().split(toStringAndClose(in));
+      return Splitter.onPattern("\r?\n").omitEmptyStrings().split(toStringAndClose(in));
```

Comment:
In fact, "remotely possible" - I have seen that case with jclouds before. The repo specifically has to use core.autocrlf = false on windows or tests fail, and the code usually splits new lines on \n. Which is ok - just does not seem to be enough in this specific case.